### PR TITLE
Update platformData.ts

### PR DESCRIPTION
**Pull request by pratmit**
Updated roadmap

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: src/components/Roadmap/data/platformData.ts [hunk 1]
```diff
@@ -34,11 +34,23 @@ export const platformData: Horizon = {
           "Customers can request to be migrated to a different SaaS cluster",
       },
       {
-        tag: [{ value: "Delegate" }],
+        tag: [{ value: "Notify" }],
         title: "Centralized Notification system",
         description:
           "Centralized notification system to configure notifications on Delegate",
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:40]:
```suggestion
          "Centralized notification system to configure delegate notifications",
```

File: src/components/Roadmap/data/platformData.ts [hunk 2]
```diff
@@ -54,6 +66,21 @@ export const platformData: Horizon = {
         title: "Priority Queue for Pipeline Execution",
         description: "",
       },
+      {
+        tag: [],
+        title: "Audit Trails",
+        description: "Send the audit events to Splunk and GCS bucket",
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:72]:
```suggestion
        description: "Send audit events to Splunk and GCS bucket",
```

File: src/components/Roadmap/data/platformData.ts [hunk 3]
```diff
@@ -54,6 +66,21 @@ export const platformData: Horizon = {
...
+      },
+      {
+        tag: [],
+        title: "Custom banners",
+        description: "Ability to display custom banners in Harness UI",
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:77]:
```suggestion
        description: "Display custom banners in the Harness UI",
```

File: src/components/Roadmap/data/platformData.ts [hunk 4]
```diff
@@ -54,6 +66,21 @@ export const platformData: Horizon = {
...
+      },
+       {
+        tag: [{ value: "Delegate" }],
+        title: "Restriction on usage of Delegate",
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:81]:
```suggestion
        title: "Restrict delegate usage",
```

File: src/components/Roadmap/data/platformData.ts [hunk 5]
```diff
@@ -54,6 +66,21 @@ export const platformData: Horizon = {
...
+        description: "Ability to restrict the usage of delegate across Account/Org/Project and Environments",
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:82]:
```suggestion
        description: "Ability to restrict delegate usage across Accounts/Orgs/Projects and Environments",
```

File: src/components/Roadmap/data/platformData.ts [hunk 6]
```diff
@@ -70,11 +97,29 @@ export const platformData: Horizon = {
         description:
           "Enable complex flows such as parallel execution, stage grouping, etc.",
       },
+      {
+        tag: [{ value: "Secure" }],
```
Comment (brian-f-harness) [inline src/components/Roadmap/data/platformData.ts:101]: Do we want to remove this (OIDC Support) since it's under `What has been released `now?

**Review by brian-f-harness — COMMENTED** [review]
Added a few minor suggestions

File: src/components/Roadmap/data/platformData.ts (hunk 6 above)
Comment (pratmit) [reply src/components/Roadmap/data/platformData.ts:101]: OIDC has different swimlanes, it is released for GCP connector but in the plan for others (secret managers)

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Pull request merged**